generic/tests/cfg/netperf.cfg

```diff
@@ -125,6 +130,13 @@
                         config_cmds = driver_verifier_query
                         driver_verifier_query = "verifier /querysettings"
                         reboot_after_config = yes
+                        install_vioprot_cmd = "netcfg -v -l %s -c p -i VIOPROT"
+                        cdroms += " virtio winutils"
+                        virtio_win_media_type = iso
+                        driver_name = netkvm VIOPROT
```

Comment:
Do you need to add VF driver?

Reply:
> Do you need to add VF driver?

Yes, we need add VF driver, but it needs download drivers from intel or other company who support SRIOV. currently, we can only manually installed vf drivers before tesing, so this part, I will research future about how to surpport it.